datacapturing/src/cyface/java/de/cyface/datacapturing/CyfaceDataCapturingService.java

```diff
@@ -108,4 +119,50 @@ public void startWifiSurveyor() throws SetupException {
             throw new SetupException(e);
         }
     }
+
+    /**
+     * For documentation see {@link DataCapturingService#start(DataCapturingListener, Vehicle, StartUpFinishedHandler)}.
+     * <p>
+     * This wrapper was requested by RM to avoid an unrecoverable state after the app crashed with an
+     * un{@link MeasurementStatus#FINISHED} {@link Measurement}.
+     * <p>
+     * This wrapper deletes "dead" {@link MeasurementStatus#OPEN} measurements because the {@link Point3d} counts
+     * was lost during app crash. "Dead" {@link MeasurementStatus#PAUSED} measurements are marked as {@code FINISHED}.
+     *
```

Comment:
See the Documentation above and/or the code below. This is the "fast-forward" solution.
We should inform RM or Klimabündnis about this and if they want that OPEN dead measurements (happen when the app crashes curing capturing) should not be deleted but recovered then we need to add a task for this for V3.

a) ether drop the sensor data in this case and finish the measurement
b) try to count the points in the sensor data - I don't think this is a good idea

Reply:
@muthenberg I implemented a) now.